src/main/java/org/sigmah/client/ui/presenter/orgunit/OrgUnitCalendarPresenter.java

```diff
@@ -113,10 +112,19 @@ public void onBind() {
 	 * {@inheritDoc}
 	 */
 	@Override
-	public void onPageRequest(final PageRequest request) {
-		final Map<Integer, Integer> calendars = new HashMap<Integer, Integer>();
-		calendars.put(1, getOrgUnit().getId());
-		calendars.put(2, getOrgUnit().getCalendarId());
+	   public void onPageRequest(final PageRequest request) {
+        final Map<Integer, Integer> calendars = new HashMap<Integer, Integer>();
```

Comment:
@spMohanty  & @osarrat  Organization calendar integration is done only in one way.That means parent calendar contains child entries while child does not contain any parent entry.Is this okay or should it be both way ?